demo_unicycle_tracking_path_jjc.py

```diff
@@ -0,0 +1,106 @@
+import sys
```

Comment:
We should talk about this demo, take a look at this 

https://www.sciencedirect.com/science/article/pii/S2405896317318748